net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf

```diff
@@ -60,6 +60,12 @@ router bgp {{ OPNsense.quagga.bgp.asnumber }}
 {%         if 'password' in neighbor and neighbor.password != '' %}
  neighbor {{ neighbor.address }} password {{ neighbor.password }}
 {%         endif %}
+{%         if 'weight' in neighbor and neighbor.weight != '' %}
+ neighbor {{ neighbor.address }} weight {{ neighbor.weight }}
+{%         endif %}
+{%     if 'disable_connected_check' in neighbor and neighbor.disable_connected_check == '1' %}
```

Comment:
Indent looks strange on a mobile :)

Reply:
you're right, it seems to be off a bit, I'll change and merge.